### PR TITLE
added TVA calculation

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,13 +191,19 @@
   <p>
     La réception d'un <em>smartphone</em> acheté dans un pays de l'Union européenne est déjà soumise à la TVA (taux en vigueur dans le pays d'expédition) et n'est pas assujettie aux droits de douane.
   </p>
+  <p>
+    L'achat de ce <em>smartphone</em> vous coutera donc <strong>{calculator.total} Euros</strong> au total.
```
Comment (davidbgk) [inline index.html:195]: `coûtera` + `euros` (-`donc`?)

File: index.html [hunk 2]
```diff
@@ -273,6 +307,11 @@
     })
   }
 
+  function submitAmount(form) {
+    calculator.amount = parseInt(form.querySelector('[name=amount]').value, 10)
+  }
+
+  calculator.init()
```
Comment (davidbgk) [inline index.html:314]: Does it reset the calculator when the user start over? I don't get where it's done.

File: index.html (hunk 1 above)
Comment (vinyll) [inline index.html:195]: Coutera : http://www.orthographe-recommandee.info/regles4.htm

Comment (davidbgk) [inline index.html:195]: Fair enough :)

File: index.html (hunk 2 above)
Comment (vinyll) [inline index.html:314]: I override the value on the next round, yet does not reset for now not to complexify the code.
Code suggestions are welcome though ;)

**Pull request merged**